_data/modules/thirdparty.yaml

```diff
@@ -89,6 +89,14 @@
   license: MIT
   maven:
     url: http://mvnrepository.com/artifact/ru.vyarus/dropwizard-guicey
+- name: kafka-dropwizard-reporter
+  url: https://github.com/SimpleFinance/kafka-dropwizard-reporter
+  description: Register built-in Kafka client metrics to Dropwizard Metrics
+  dropwizard: 1.0.0, 0.7+
```

Comment:
I'm not quite sure how to express version compatibility here. This module only requires dropwizard-metrics 3.0+, so is compatible with dropwizard 0.7 to the latest. Is there a preferred pattern?